build.gradle

```
@@ -3,6 +3,7 @@ buildscript {
     ext.kotlin_version = '1.4.32'
 
     repositories {
+        maven { url "https://a8c-libs.s3.amazonaws.com/android" }
```

```suggestion
        maven {
            url 'https://a8c-libs.s3.amazonaws.com/android' 
            content {
                includeGroup "com.automattic.android"
                includeGroup "com.automattic.android.publish-to-s3"
            }
        }
```

We have been using a more strict syntax for Gradle repositories, so it'd be great if you can replace it with this suggestion.

---

Good stuff – I've applied this suggestion!